### PR TITLE
[#81] Update docs

### LINKED ISSUE #81: Document Postgres setup
Doing `stack build` I get the following error:

```
--  While building package postgresql-libpq-0.9.4.2 using:
      /tmp/stack-9b8112fa643e992a/postgresql-libpq-0.9.4.2/.stack-work/dist/x86_64-linux/Cabal-2.4.0.1/setup/setup --builddir=.stack-work/dist/x86_64-linux/Cabal-2.4.0.1 configure --with-ghc=/home/simon/.stack/programs/x86_64-linux/ghc-8.6.5/bin/ghc --with-ghc-pkg=/home/simon/.stack/programs/x86_64-linux/ghc-8.6.5/bin/ghc-pkg --user --package-db=clear --package-db=global --package-db=/home/simon/.stack/snapshots/x86_64-linux/lts-13.26/8.6.5/pkgdb --libdir=/home/simon/.stack/snapshots/x86_64-linux/lts-13.26/8.6.5/lib --bindir=/home/simon/.stack/snapshots/x86_64-linux/lts-13.26/8.6.5/bin --datadir=/home/simon/.stack/snapshots/x86_64-linux/lts-13.26/8.6.5/share --libexecdir=/home/simon/.stack/snapshots/x86_64-linux/lts-13.26/8.6.5/libexec --sysconfdir=/home/simon/.stack/snapshots/x86_64-linux/lts-13.26/8.6.5/etc --docdir=/home/simon/.stack/snapshots/x86_64-linux/lts-13.26/8.6.5/doc/postgresql-libpq-0.9.4.2 --htmldir=/home/simon/.stack/snapshots/x86_64-linux/lts-13.26/8.6.5/doc/postgresql-libpq-0.9.4.2 --haddockdir=/home/simon/.stack/snapshots/x86_64-linux/lts-13.26/8.6.5/doc/postgresql-libpq-0.9.4.2 --dependency=Cabal=Cabal-2.4.1.0-9MZFDeNrcJI10bcroa6pq8 --dependency=base=base-4.12.0.0 --dependency=bytestring=bytestring-0.10.8.2 --dependency=unix=unix-2.7.2.2
    Process exited with code: ExitFailure 1
```

I spent a little time figuring out that on Ubuntu I need to `apt install libpq-dev` to compile this.

For running Postgres, I need to

```
$ sudo apt install postgresql postgresql-contrib
$ sudo service postgres start
$ sudo -u postgres psql
postgres=# create database "issue-wanted";
postgres=# create user simon;
postgres=# grant all privileges on database "issue-wanted" to simon;
```

I [modified `pg_hba.conf`](https://dba.stackexchange.com/questions/101280/how-to-handle-user-with-no-password-in-postgresql) with the lines

```
local all simon          trust
host all simon 0.0.0.0/0 trust
```

(This is a little unsafe, I realize, but for some reason `127.0.0.1/8` didn't cut it.)

and changed `user=simon` in config.toml and added `listen_address = '127.0.0.1'` in `/etc/postgresql/10/main/postgresql.conf`.

I then restarted Postgres and initialized the database manually:

```
$ sudo service postgres restart
$ psql issue-wanted < sql/schema.sql
$ psql issue-wanted < sql/seed.sql
$ stack exec issue-wanted
```

At this point the `/issues` endpoint is responding positively!

Perhaps we should document some of this in `README.md`?

### PR DESCRIPTION
**Pull request by rashadg1030**
Resolves #81

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,45 @@ encourage more programmers to become a part of the Haskell community.
...
+For the project to build, you will need to have `libpq-dev` installed on you computer. You can install it by running the command `sudo apt install libpq-dev`.
+
+You will also need to setup Postgres on your computer. Here are the instructions for doing so:
+
+1. Run the folowing commands with `<username>` replaced by a username of your choice.
+
+   ```
+   $ sudo apt install postgresql postgresql-contrib
+   $ sudo service postgres start
+   $ sudo -u postgres psql
```
Comment (chshersh) [inline README.md]: @rashadg1030 Thanks for writing all these instructions. But we actually don't need all of that. Our DB is shipped within `docker` container. It's already configured and works on both Linux and macOS (not sure about Windows). So developers don't need to install, configure and run the database itself in order to build, test or run `issue-wanted`. All you need is to have `docker` installed and then just run:

* `make postgres`

This command will run the database. To build `issue-wanted` you still need `libpq-dev`, so you can provide a command to install it. But that's all. So, prerequisites for `issue-wanted`:

* `ghc`
* `cabal` or `stack`
* `docker`
* `libpq-dev` (`sudo apt install libpq-dev`)

File: README.md [hunk 2]
```diff
@@ -13,41 +13,14 @@ encourage more programmers to become a part of the Haskell community.
 
 ## Prerequisites (what you need to have locally)
 
-For the project to build, you will need to have `libpq-dev` installed on you computer. You can install it by running the command `sudo apt install libpq-dev`.
+You will need to have the following installed on your system in order to build and test `issue-wanted`. Click on the links to learn how to install each one:
 
-You will also need to setup Postgres on your computer. Here are the instructions for doing so:
+1. [ghc](https://downloads.haskell.org/~ghc/6.8.3/docs/html/users_guide/installing-bin-distrib.html)
```
Comment (chshersh) [inline README.md]: Wow, `6.8.3` is a very old GHC version. Let's link to `ghcup` instead:

* https://www.haskell.org/ghcup/

Comment (rashadg1030) [reply README.md]: Yes, good call. Thank you!

**Comment by rashadg1030** [comment]
Changed link to ghc installation instructions.

**Pull request merged**